WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -248,17 +248,25 @@ def update_events
     end
   end
 
+  def get_nearby_competitions(competition)
+    nearby_competitions = competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING)[0, 10]
+    nearby_competitions.select!(&:isConfirmed?) unless current_user.can_view_hidden_competitions?
+    nearby_competitions || []
```

Comment:
I think you may be right. It was probably failing due to something else, so we talked and thought it may be nil. I'll test without it (and with no nearby comps as an organizer) and report back.